src/qt/radiobut.cpp

```diff
@@ -59,3 +70,40 @@ QWidget *wxRadioButton::GetHandle() const
 {
     return m_qtRadioButton;
 }
+
+void wxRadioButton::CreateAndJoinNewGroup()
+{
+    QButtonGroup* qtButtonGroup = new QButtonGroup( GetHandle() );
```

Comment:
AFAICS this is still being leaked. Am I missing something? I just don't understand why wasn't this made a member of `wxRadioButton` (as we seemed to have agreed to do it when discussing a previous version)...

Reply:
It should be obvious for people only knowing wxW too. A 1-line explaining the ownership would indeed be nice.